tailfs/birthtiming.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package tailfs
+
+import (
+	"context"
+	"io/fs"
+	"os"
+	"time"
+
+	"github.com/djherbis/times"
+	"github.com/tailscale/xnet/webdav"
```

Comment:
Not for this PR specifically, but do we have any plans for getting away from having to maintain a fork of this package? Do we think it's possible to get our changes upstreamed so we don't have to maintain the fork indefinitely?

I don't think it's a problem to have the fork, but over time it's easy for security issues to creep in via bit rot on forks, so I'm hoping this is something we can keep on our radar to maybe upstream as soon as is practical,.

Reply:
Yes, I tried to make the change in a way that's hopefully non-controversial, and there's a task in https://github.com/tailscale/corp/issues/16827 to work on getting this upstreamed.